aicssegmentation/workflow/batch_workflow.py

```diff
@@ -146,11 +146,13 @@ def write_log_file_summary(self):
         Write a log file to the output folder.
         """
         if self._processed_files == 0:
-            report = "There were no files to process in the input directory"
+            report = f"There were no files to process in the input directory to process \n " \
+                     f"Using the Workflow: {self._workflow_definition.name}"
```

Comment:
Just checking, would this also work when the user uses a workflow json file? Is `self._workflow_definition.name` the file name in that case?

Reply:
I tested with both workflow json as well as workflow definition- in both cases the name is saved to that private variable